bucket/kryptor.json

```
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://www.kryptor.co.uk/",
+    "description": "A simple, modern, and secure encryption and signing tool.",
+    "version": "3.1.0",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/samuel-lucas6/Kryptor/releases/download/v3.1.0/kryptor-windows.zip",
+            "hash": "c6142e7b987c390ef9bc097273ec9e4284d901619e3f74c97a6f4e15430b7da6"
+        }
+    },
+    "bin": "kryptor.exe",
+    "checkver": {
+        "github": "https://github.com/samuel-lucas6/Kryptor"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/samuel-lucas6/Kryptor/releases/download/v$version/kryptor-windows.zip",
+                "hash": {
+                    "url": "$url.DIGEST",
+                    "regex": "([a-fA-F0-9]{32,128})[\\x20\t]+.*`$basename(?:[\\x20\t]+\\d+)?"

```

```suggestion
                    "url": "$url.DIGEST"
```

Actually, you don't need regex here at all